lib/parser/interfaces.ts

```diff
@@ -12,7 +12,7 @@ export interface IMayHaveLabel {
   labelLinks: OLabelReference[];
 }
 export function implementsIHasLabel(obj: ObjectBase): obj is ObjectBase & IHasLabel {
-  return (obj as ObjectBase & IHasLabel).label !== undefined && Array.isArray((obj as ObjectBase & IHasLabel).labelLinks);
+  return (obj as ObjectBase & Partial<IHasLabel>).label !== undefined && Array.isArray((obj as ObjectBase & IHasLabel).labelLinks);
```

Comment:
```suggestion
  return (obj as ObjectBase & IHasLabel).label !== undefined && Array.isArray((obj as ObjectBase & IHasLabel).labelLinks);
```
partial is not required here and in the whole file. Undo these changes

Reply:
It actually makes more sense with Partial though.... If it is not partial the check does not make sense...